cpp/src/parquet/file_writer.cc

```diff
@@ -310,6 +311,7 @@ class FileSerializer : public ParquetFileWriter::Contents {
         num_row_groups_(0),
         num_rows_(0),
         metadata_(FileMetaDataBuilder::Make(&schema_, properties, key_value_metadata)) {
+    file_metadata_.reset();
```

Comment:
Is this needed (it should be null at initialization time)?

Reply:
Removed.